python/semantic_kernel/connectors/memory/milvus/milvus_memory_store.py

```diff
@@ -0,0 +1,470 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from logging import Logger
+from typing import List, Optional, Tuple
+
+from numpy import array, expand_dims, ndarray
+from pymilvus.milvus_client import milvus_client
+
+from semantic_kernel.memory.memory_record import MemoryRecord
+from semantic_kernel.memory.memory_store_base import MemoryStoreBase
+from semantic_kernel.utils.null_logger import NullLogger
+
+
+def memoryrecord_to_milvus_dict(mem: MemoryRecord) -> dict:
+    """Convert a memoryrecord into a dict.
+    Args:
+        mem (MemoryRecord): MemoryRecord to convert.
+
+    Returns:
+        dict: Dict result.
+    """
+    ret_dict = {}
+    # Grab all the class vars
+    for key, val in vars(mem).items():
+        if val is not None:
+            # Remove underscore
+            ret_dict[key[1:]] = val
+    return ret_dict
+
+
+def milvus_dict_to_memoryrecord(milvus_dict: dict) -> MemoryRecord:
+    """Convert Milvus search result dict into MemoryRecord.
+
+    Args:
+        milvus_dict (dict): Search hit
+
+    Returns:
+        MemoryRecord
+    """
+    # Embedding needs conversion to numpy array
+    embedding = milvus_dict.get("embedding", None)
+    if embedding is not None:
+        embedding = array(embedding)
+    return MemoryRecord(
+        is_reference=milvus_dict.get("is_reference", None),
+        external_source_name=milvus_dict.get("external_source_name", None),
+        id=milvus_dict.get("id", None),
+        description=milvus_dict.get("description", None),
+        text=milvus_dict.get("text", None),
+        additional_metadata=milvus_dict.get("additional_metadata", None),
+        embedding=embedding,
+        key=milvus_dict.get("key", None),
+        timestamp=milvus_dict.get("timestamp", None),
```

Comment:
What representation actually gets saved to Milvus, given that it [doesn't seem to support](https://milvus.io/docs/create_collection.md) any sort of date/time type?

Reply:
my understanding is that timestamp resolves to a string